user_manual/groupware/contacts.rst

```diff
@@ -182,3 +182,7 @@ Circle options
 ~~~~~~~~~~~~~~
 
 Various self-explanatory options are available to configure a circle, to manage invites and membership, visibility of the circle, allowance of other circle membership and password protection.
+
+Shared items
+~~~~~~~~~~~~
+The contact app supports shared item starting from version 5.5.0. Media, calendar events, talk chat room between you and another contact or shared deck cards, will be shown on the contact details. You need to have the app activated in order to see the shared items of that app. Right now, we only support the shared items between 2 contacts.
```

Comment:
* The user can't enable an app. Drop the note about enabled app.
* Use https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded to render when the feature was added.
* Add note that this only works for contacts in the *system address book*